### DISCUSSION TITLE
What does the NML tag stand for? It is not in the glossary.

### DISCUSSION THREAD

**Discussion by kyleiwaniec** [explosion/spaCy#11484] [Help: Other Questions] [answered]
## Which page or section is this issue related to?
https://github.com/explosion/spaCy/blob/master/spacy/glossary.py

I am trying to understand the parse tree of the following sentence fragment:

                                                                       
```
import nltk
from nltk import Tree, ParentedTree
import benepar, spacy
nlp = spacy.load('en_core_web_md')
nlp.add_pipe('benepar', config={'model': 'benepar_en3'})

doc = nlp("WHO Director - General Tedros Adhanom Ghebreyesus told reporters")
sent = list(doc.sents)[0]
parse_tree = ParentedTree.fromstring('(' + sent._.parse_string + ')')
print(parse_tree.pretty_print())
```
```

                            |                                           
                            S                                          
                      ______|______________________________             
                     NP                                    VP          
        _____________|________________________         ____|______      
      NML                   |       |         |       |           NP   
  _____|_____________       |       |         |       |           |     
NNP   NNP    HYPH   NNP    NNP     NNP       NNP     VBD         NNS   
 |     |      |      |      |       |         |       |           |     
WHO Director  -   General Tedros Adhanom Ghebreyesus told     reporters
```

```
from spacy import glossary
tag_name = 'NML'
glossary.explain(tag_name)
```

However, I am not able to find the meaning of the acronym NML. It is not listed in the glossary. I'm not sure where else to look. Could someone please advise what NML stands for or where I can find it?

Thank you!

**Comment by polm** [comment, marked as answer]
`benepar` is a third-party parser, so not all the terms used in it are in the glossary. Based on [Penn Treebank docs](https://catalog.ldc.upenn.edu/docs/LDC2011T03/treebank/english-treebank-guidelines-addendum.pdf), it looks like "NML" stands for "nominal modifiers". To quote from there:

> We use the NML node label to mark nominal subconstituents that do not follow the default
right-branching structure. Any two or more non-final elements that form a constituent are bound
together by NML. 

```
(NP (NML Cytochrome P450)
  isoenzyme)
(NP selective
  (NML seratonin reuptake)
  inhibitor)
(NP (NML gel electrophoresis)
  analysis)
(NP (NML 5 nM)
  tetrachlorobiphenyl)
(NP (NML human liver tumor)
  analysis)
```
